package.json

```diff
@@ -22,6 +22,7 @@
         "react": "^16.8.6",
         "react-dom": "^16.8.6",
         "react-flexbox-grid": "^2.1.2",
+        "react-markdown": "^4.1.0",
```

Comment:
How much did it say the build size was before/after react-markdown was added?

Reply:
I'm honestly not sure how to check this, it didn't take long to add tho.